src/features/chats/hooks/use-chat-socket.ts

```
@@ -0,0 +1,98 @@
+      if (clientRef.current) {
+        console.log('[STOMP] Deactivating client')
+        clientRef.current.deactivate()
+        clientRef.current = null
+      }
+    }
+  }, [chatId, onMessage])
+
+  const sendMessage = (message: OutgoingMessage) => {
+    const client = clientRef.current
+
+    if (client && client.connected) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)